<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>homeClass</class>
 <widget class="QWidget" name="homeClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>home</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#widget,#widget_2,#widget_3,#widget_4,#widget_5{
	border-radius:8;
}</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>19</y>
     <width>211</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>243</x>
     <y>19</y>
     <width>211</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>140</y>
     <width>211</width>
     <height>60</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>243</x>
     <y>140</y>
     <width>211</width>
     <height>60</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_5" native="true">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>214</y>
     <width>431</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../res/res.qrc"/>
 </resources>
 <connections/>
</ui>
